docs/source/tutorial.rst

```diff
@@ -72,9 +59,93 @@ Open up a terminal and run the following:
       C:\> git clone https://github.com/hyperledger/sawtooth-core.git
       --config core.autocrlf=false
 
+Docker Instructions
+===================
+
+Prerequisites
+-------------
+
+The following tools are required:
+
+* `Docker Engine <https://docs.docker.com/engine/installation/>`_ (17.03.0-ce
+  or later)
+* `Docker Compose <https://docs.docker.com/compose/install/>`_ (Linux only)
 
 Environment Startup
-===================
+-------------------
+
+To start up the environment, run:
+
+.. code-block:: console
+
+  % cd sawtooth-core
+  % docker-compose -f docker/compose/sawtooth-demo.yaml up
+
+Downloading the docker images that comprise the Sawtooth Lake demo
+environment can take serveral minutes. Once you see the containers
+registering and creating intial blocks you can move on to the next step.
+
+.. code-block:: console
+
+  Attaching to compose_validator_1, compose_tp_xo_python_1, compose_client_1, compose_tp_intkey_python_1, compose_tp_config_1, compose_rest_api_1
+  validator_1         | writing file: /etc/sawtooth/keys/validator.wif
+  validator_1         | writing file: /etc/sawtooth/keys/validator.addr
+  validator_1         | Generating /var/lib/sawtooth/genesis.batch
+  tp_xo_python_1      | [19:03:47 DEBUG   selector_events] Using selector: ZMQSelector
+  validator_1         | [19:03:47.537 INFO     path] Skipping path loading from non-existent config file: /etc/sawtooth/path.toml
+  tp_xo_python_1      | [19:03:47 INFO    core] register attempt: OK
+  validator_1         | [19:03:47.538 INFO     cli] config [path]: config_dir = "/etc/sawtooth"
+  tp_intkey_python_1  | [19:03:47 DEBUG   selector_events] Using selector: ZMQSelector
+  validator_1         | [19:03:47.538 INFO     cli] config [path]: key_dir = "/etc/sawtooth/keys"
+  tp_intkey_python_1  | [19:03:47 INFO    core] register attempt: OK
+
+Open a new terminal so we can connect to the client container:
+
+.. code-block:: console
+
+  % docker exec -it compose_client_1 bash
+
+Your environment is ready! Continue on to `Multi-language support for transaction processors`_.
+
+Resetting The Environment
+-------------------------
+
+If the environment needs to be reset for any reason, it can be returned to
+the default state by logging out of the client container, then pressing
+CTRL-c from the window where you originally ran docker-compose. Once the
+containers have all shut down:
```

Comment:
Do what? Suggestion:
Once the containers have all shut down, enter the command 'docker-compose -f sawtooth-demo.yaml down':